test/test_rdf_writer.py

```diff
@@ -116,6 +116,17 @@ def test_adding_values(self):
         w.convert_to_rdf()
         self.assertEqual(len(list(w.g.subjects(predicate=RDF.li, object=Literal("val")))), 3)
 
+    def test_section_subclass(self):
```

Comment:
Add tests for
- a Section that has a non subclassed type.
- a Section that has type None.

Reply:
I have already checked that in adding simple sections earlier. Since the subclass sections functions are invoked only when type is subclassed
`if fmt._name == "section":`
`      s = self._get_section_subclass(e)`
`      u = s if s else fmt.rdf_type()`
`      self.g.add((curr_node, RDF.type, URIRef(u)))`
there is no need to check non subclassed and None types in the test.